### ISSUE TITLE
Train Logistic Regression : Data Preparation for Machine Learning

### ISSUE THREAD

**Issue by kazourak** [farinaleo/ft_dslr#21]
# Data Preparation

## Description

As a user, I want to perform several tasks on a CSV file in order to prepare the data for logistic regression modeling.

## Acceptance Criteria:
- [x]  Load the CSV file containing the data.
- [x] Remove columns that are not relevant for analysis.
- [x] Encode categorical variables to make them usable for modeling.
- [x] Check and handle missing values in the data to avoid biases.
- [x] Separate the features from the target variable.
- [x] Split the data into two distinct sets: a training set and a test set, to evaluate the model's performance.

**Comment by kazourak** [comment]
My proposal : 
- Load CSV file to a pandas DataFrame
- Remove Index, First_name and Last_name and Birthday columns
- Encode Best_hand and Hogwarts_house
- Fill missing values with the mean of their column
- The split function returns 2 dataframes and 2 series, considering a percentage. The percentage indicates how much data will be used for training; the rest will be for testing. One dataframe is for the training set, and another dataframe is for the test set. The series represent the expected house values.
